From: Sven Roederer <devel-sven@geroedel.de>
Date: Sat, 17 Apr 2021 15:51:08 +0200
Subject: sdk: unset BINARY_FOLDER and DOWNLOAD_FOLDER in SDK archive

Setting these folder during buildtime will embedd their location in the .config file.
This results in trying to use the same subfolders on the (other) machine running the
imagebuilder, where these might be not accessible at all.

Signed-off-by: Sven Roederer <devel-sven@geroedel.de>

diff --git a/target/sdk/Makefile b/target/sdk/Makefile
index 0606621192c7508fc7a470eff0173dc62ff70ded..d85c17b613e518bffac996932e0fd090511660d6 100644
--- a/target/sdk/Makefile
+++ b/target/sdk/Makefile
@@ -159,6 +159,8 @@ $(BIN_DIR)/$(SDK_NAME).tar.xz: clean
 	$(SED) 's,^# REVISION:=.*,REVISION:=$(REVISION),g' $(SDK_BUILD_DIR)/include/version.mk
 	$(SED) 's,^# SOURCE_DATE_EPOCH:=.*,SOURCE_DATE_EPOCH:=$(SOURCE_DATE_EPOCH),g' $(SDK_BUILD_DIR)/include/version.mk
 	$(SED) '/LINUX_VERMAGIC:=/ { s,unknown,$(LINUX_VERMAGIC),g }' $(SDK_BUILD_DIR)/include/kernel.mk
+	$(SED) 's,default "$(CONFIG_BINARY_FOLDER)",default "",' $(SDK_BUILD_DIR)/Config.in
+	$(SED) 's,default "$(CONFIG_DOWNLOAD_FOLDER)",default "",' $(SDK_BUILD_DIR)/Config.in
 	find $(SDK_BUILD_DIR) -name .git | $(XARGS) rm -rf
 	find $(SDK_BUILD_DIR) -name .svn | $(XARGS) rm -rf
 	find $(SDK_BUILD_DIR) -name CVS | $(XARGS) rm -rf
